include/mgmt/hawkbit.h

```diff
@@ -5,8 +5,8 @@
  */
 
 /**
- * @brief Hawkbit Firmware Over-the-Air for Zephyr Project.
- * @defgroup hawkbit Hawkbit Firmware Over-the-Air
+ * @brief hawkBit Firmware Over-the-Air for Zephyr Project.
+ * @defgroup hawkBit Firmware Over-the-Air
```

Comment:
```suggestion
 * @defgroup hawkbit hawkBit Firmware Over-the-Air
```

Reply:
Ah I thought that was a typo, no worries, fixed!